### ISSUE TITLE
Namespacing not working in v0.8.4

### ISSUE THREAD

**Issue by edbentinck** [bobthecow/psysh#373]
I am using Psy Shell via Laravel Tinker. However, in v0.8.4 `namespace` doesn't seem to take effect. I just manually downgraded to v0.8.3 and namespace works fine.

This is the expected behaviour, as seen in v0.8.3:
```
Psy Shell v0.8.3 (PHP 5.6.30 — cli) by Justin Hileman
>>> namespace App;
=> null
>>> User::first();
=> App\User {...}
```

The actual behaviour in v0.8.4:
```
Psy Shell v0.8.4 (PHP 5.6.30 — cli) by Justin Hileman
>>> namespace App;
>>> User::first();
PHP Fatal error:  Class 'User' not found in eval()'d code on line 1
>>> \App\User::first();
=> App\User {...}
```

**Comment by bobthecow** [comment]
Thanks for the report. I'll check it out!

**Comment by bobthecow** [comment]
Here you go!

<img width="762" alt="screen shot 2017-05-16 at 9 59 38 pm" src="https://cloud.githubusercontent.com/assets/53660/26139092/05d8fb02-3a83-11e7-845a-8abcd5e63312.png">

This is fixed in develop. I'll get a release cut shortly.

**Comment by edbentinck** [comment]
Awesome! Thanks a bunch :) Super quick response.

**Comment by bobthecow** [comment]
This has been released in v0.8.5. Thanks again for reporting :)
